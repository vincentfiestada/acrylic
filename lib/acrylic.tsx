/**
 * Re-export all components in one place
 */
export { default as IProps } from "./components/IProps";
export { default as IEventProps } from "./components/IEventProps";

/**
 * Layout Components
 */
export { default as ILayoutProps } from "./components/layout/ILayoutProps";
export { default as Section } from "./components/layout/Section";
export { default as Column } from "./components/layout/Column";
export { default as Row } from "./components/layout/Row";
export { default as Alert } from "./components/layout/Alert";
export { default as Toolbar } from "./components/layout/Toolbar";

/**
 * Typography Components
 */
export { default as ITextProps } from "./components/typography/ITextProps";
export { default as Title } from "./components/typography/Title";
export { default as Text } from "./components/typography/Text";
export { default as Icon } from "./components/typography/Icon";

/**
 * Control Components
 */
export { default as Button } from "./components/controls/Button";
export { default as IconButton } from "./components/controls/IconButton";

/**
 * Variants
 */
export { default as Variant } from "./components/Variant";
