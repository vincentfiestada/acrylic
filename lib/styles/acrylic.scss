@import "normalize.css";
@import "typography";
@import "controls";
