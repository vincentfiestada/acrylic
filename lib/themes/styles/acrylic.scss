@import "normalize.css";
@import "layout";
@import "typography";
@import "icons";
@import "controls";
@import "animation";
