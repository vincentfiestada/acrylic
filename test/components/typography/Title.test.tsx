import * as React from "react";
import { Title } from "../../../lib/acrylic";

describe("<Title />", () => {
	it("Should match the latest snapshots", () => {
		expect(
			<React.Fragment>
				<Title level={1}>Chapter</Title>
				<Title level={2}>Chapter</Title>
				<Title level={3}>Chapter</Title>
				<Title level={4}>Chapter</Title>
				<Title level={5}>Chapter</Title>
				<Title level={6}>Chapter</Title>
				<Title inline>Chapter</Title>
				<Title bold>Chapter</Title>
				<Title italic>Chapter</Title>
				<Title underlined>Chapter</Title>
			</React.Fragment>
		).toMatchSnapshot();
	});
});
