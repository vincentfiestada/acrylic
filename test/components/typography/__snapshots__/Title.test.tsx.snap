// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Title /> Should match the latest snapshots 1`] = `
Array [
  <Title
    level={1}
  >
    <h1
      className="acr-title"
    >
      Chapter
    </h1>
  </Title>,
  <Title
    level={2}
  >
    <h2
      className="acr-title"
    >
      Chapter
    </h2>
  </Title>,
  <Title
    level={3}
  >
    <h3
      className="acr-title"
    >
      Chapter
    </h3>
  </Title>,
  <Title
    level={4}
  >
    <h4
      className="acr-title"
    >
      Chapter
    </h4>
  </Title>,
  <Title
    level={5}
  >
    <h5
      className="acr-title"
    >
      Chapter
    </h5>
  </Title>,
  <Title
    level={6}
  >
    <h6
      className="acr-title"
    >
      Chapter
    </h6>
  </Title>,
  <Title
    inline={true}
  >
    <h6
      className="acr-title acr-inline"
    >
      Chapter
    </h6>
  </Title>,
  <Title
    bold={true}
  >
    <h6
      className="acr-title acr-bold"
    >
      Chapter
    </h6>
  </Title>,
  <Title
    italic={true}
  >
    <h6
      className="acr-title acr-italic"
    >
      Chapter
    </h6>
  </Title>,
  <Title
    underlined={true}
  >
    <h6
      className="acr-title acr-underlined"
    >
      Chapter
    </h6>
  </Title>,
]
`;
